prototype2/common/src/world.rs.in

```diff
@@ -69,7 +68,24 @@ pub struct ClientWorld {
   pub disabled: HashSet<Entity>
 }
 
-pub type Entity = Uuid;
+pub struct ClientWorld2 {
+  #[allow(dead_code)]
```

Comment:
I'm assuming these are temporary unless they are for an external api. 
I wonder if rustc is baller enough to tell you if you don't need this warning suppressor anymore.